packages/overlays/src/localOverlaysStyle.js

```diff
@@ -0,0 +1,41 @@
+import { css } from '@lion/core';
```

Comment:
Would it also be possible to split up between functional (always applied, but overridable) and theme styling?

We do this basically for every component. (In this case the backdrop would be just a webcomponent that you can extend and supply yourself). The backdrop would then also become it's own BEM component `.backdrop` that can be shared for global and local (would also be convenient if you switch from local to overlay based on viewport)

Functional would be:
- positioning and dimensions
- display
- default z-index
- no animjation

Theme styles would involve
- background
- animations can be added in either js or css, up to developer

Reply:
I like that, but see my other comment, I think we should tackle this in a separate PR, so we have a `<lion-overlays-backdrop>` for both global and local, and then we can implement this. But we need to think about backwards compatibility with our extension layer, where people probably rely right now on overriding styles of global overlays from the document, because it's not encapsulated in a ShadowDOM at the moment :\

Reply:
If only we had a tool to find this out... :)